streaming/stream_blob.cc

```diff
@@ -179,6 +180,10 @@ future<> stream_blob_handler(replica::database& db,
                 blogger.info("stream_mutation_fragments: released (tablets)");
             });
 
+            if (db.is_in_critical_disk_utilization_mode()) {
+                throw replica::critical_disk_utilization_exception("rejected streamed file");
```

Comment:
please be more elaborate in exception string, e.g. "rejected streamed file due to critical disk utilization"